civitai-downloader/src/search.py

```
@@ -0,0 +1,259 @@
+                limit=limit + 1
+            )
+            
+            results = await self.search(params)
+            return [m for m in results if m.id != model.id][:limit]
+    
+    async def get_trending(self, time_period: str = "week", limit: int = 20) -> List[ModelInfo]:
+        """Get trending models for a time period."""
+        # The API doesn't directly support time-based trending,
+        # so we'll use most downloaded as a proxy
+        params = SearchParams(
+            sort=SortOrder.MOST_DOWNLOADED,
```

Ruff (0.12.2)

Undefined name `SortOrder`

(F821)